smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -59,6 +59,7 @@ contract MixinTransfer is
     onlyIfAlive
     onlyKeyOwnerOrApproved(_tokenId)
   {
+    require(transferFeeBasisPoints < 10000, 'KEY_TRANSFERS_DISABLED');
```

Comment:
Maybe use the constant here for consistency?

Reply:
Yeah, good idea. will make the change in `shareKey` & `transferFrom` and get this merged.